examples/tekton/01-rolebinding-share.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: shared-resource-my-share-tekton-default
+  namespace: my-csi-app-namespace
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: shared-resource-my-share-tekton
+subjects:
+  - kind: ServiceAccount
+    name: default
```

Comment:
Is this service account that the TaskRun pod runs as ?

Reply:
yes it is

and I made a conscious decision to use `default` instead of `pipeline`:

- although shared resources is still a "downstream" thing, whenever possible I try to make it as "upstream friendly" as I can
- I also had vague recollections about generally wanting to move away from the `pipeline` SA (but of course I may have incorrect or out of date recollections here)
- and of course, switching the `pipeline` here, or simply using `pipeline` in whatever version of this might land in app studio, should be straight forward

Reply:
Sounds good, Gabe!